go/vt/sqlparser/parse_test.go

```diff
@@ -607,7 +607,8 @@ var (
 		input:  "select /* binary unary */ a- -b from t",
 		output: "select /* binary unary */ a - -b from t",
 	}, {
-		input: "select /* - - */ - -b from t",
+		input:  "select /* - - */ - -b from t",
+		output: "select /* - - */ +b from t",
```

Comment:
Shouldn't `- -` just cancel each other out instead of turning into a unary plus?

Reply:
done